src/test/test_snap_mapper.cc

```diff
@@ -461,7 +461,18 @@ class MapperVerifier {
     uint32_t bits)
     : driver(driver),
       mapper(new SnapMapper(g_ceph_context, driver, mask, bits, 0, shard_id_t(1))),
-             mask(mask), bits(bits) {}
+	     mask(mask), bits(bits) {}
+
+  hobject_t create_hobject(unsigned           idx,
+			   snapid_t           snapid,
```

Comment:
nit: inconsistent indent. Let's move also the first param to a new lines to be consistent with e.g. `get_next_or_current()` in this file.